codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/GoDependency.java

```diff
@@ -52,7 +56,9 @@
     AWS_PRIVATE_PROTOCOL("dependency", "github.com/aws/aws-sdk-go-v2",
             "github.com/aws/aws-sdk-go-v2/aws/private/protocol", null, Versions.AWS_SDK),
     AWS_JSON_PROTOCOL("dependency", "github.com/aws/aws-sdk-go-v2",
-            "github.com/aws/aws-sdk-go-v2/aws/protocol/json", null, Versions.AWS_SDK);
+            "github.com/aws/aws-sdk-go-v2/aws/protocol/json", null, Versions.AWS_SDK),
+    AWS_JSON_PROTOCOL_ALIAS("dependency", "github.com/aws/aws-sdk-go-v2",
```

Comment:
This is not used

Reply:
It is used in aws/aws-sdk-go-v2#568.
Does it need to be defined in v2 repo instead?